### ISSUE TITLE
Segmentation fault when loading pretrained file

### ISSUE THREAD

**Issue by davidz123** [microsoft/unilm#181]
When loading model weights file using <model.from_pretrained>, segmentation fault error occur..

**Comment by donglixp** [comment]
Could you provide more details (such as the model name, and the command) about the error? Thanks

**Comment by Krishna-suraj** [comment]
Unable to run the finetuning process as mentioned. getting `Segmentation fault (core dumped)` in the console.
Command is  `python run_seq_labeling.py  --data_dir data \
                            --model_type layoutlm \
                            --model_name_or_path unilm/layoutlm/models/layoutlm-base-uncased \
                            --do_lower_case \
                            --max_seq_length 512 \
                            --do_train \
                            --num_train_epochs 100.0 \
                            --logging_steps 10 \
                            --save_steps -1 \
                            --output_dir unilm/layoutlm/models/fake_output/try1 \
                            --labels data/labels.txt \
                            --per_gpu_train_batch_size 1 \
                            --per_gpu_eval_batch_size 1 \
                            --fp16 --overwrite_output_dir`

**Comment by empowerszc** [comment]
@Krishna-suraj , Hi, I also meet the problem and have no idea to deal with it. Do you resolve it? Really appreciate your help!

**Comment by wyc2015fq** [comment]
@Krishna-suraj , @empowerszc, I also meet this problem. Do you resolve it?

**Comment by empowerszc** [comment]
@wyc2015fq  I don't resolve it, I am lacking of experience to debug it. I would try again when i have better understanding. If you solved this problem, look forwarding to  sharing your solution.

**Comment by elnazsn1988** [comment]
I have the same error, does anyone know why this is?

**Comment by elnazsn1988** [comment]
@Krishna-suraj @davidz123 are you guys running this on GCP or any other cloud platform by any chance? apparently (google) says it has to do with memory access....

**Comment by elnazsn1988** [comment]
@wyc2015fq @empowerszc I had same error in the latest git commit from the team, however previously had forked and not merged old code and it doesn't have this issue : to anyone else facing issue - I had previously forked old version, this version works for me: https://github.com/elnazsn1988/unilm/tree/master/layoutlm

**Comment by Krishna-suraj** [comment]
Hi, sorry for late response.This issue resolved for me after I used CUDA 10.1. I will share my stack, Python 3.6.10; torch==1.5.0 and other in layoutlm/setup.py

**Comment by elnazsn1988** [comment]
@Krishna-suraj @davidz123 @wyc2015fq @empowerszc @donglixp Update - Using pytorch 1.6 Cuda 10.1 removed all issues and allowed installation of latest commit.

**Comment by varshaneya** [comment]
Pytorch 1.6 and Cuda toolkit 10.1 solved the problem for me as well. @donglixp could you make that change in the Readme.md?

**Comment by donglixp** [comment]
Thanks for your solutions. Please feel free to submit a pull&request to the repo.

**Comment by yixuanren** [comment]
Same issue here. (It's still `pytorch==1.4.0` in Readme so far.)

With Python 3.6.12 (the latest subversion of 3.6 now), PyTorch 1.4 produces core dumped at the very beginning when running the seq_labeling example. PyTorch 1.5 fails the installation of Apex (looks like something about ninja but not sure). Finally PyTorch 1.6 works well.

(Cuda version always 10.1.243 and no other config changed)

But I'm still not sure how Readme should be changed -- after all we haven't tested every corner and output of the entire program -- so this may still up rely on the original authors for any further instructions.
